lib/caps/ICL/iHD

```diff
@@ -0,0 +1,77 @@
+###
+### Copyright (C) 2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+caps = dict(
+  decode  = dict(
+    jpeg    = dict(maxres = (8192, 8192)),
+    mpeg2   = dict(maxres = (4096, 4096)),
+    vc1     = dict(maxres = (4096, 4096)),
+    avc     = dict(maxres = (4096, 4096)),
+    hevc_8  = dict(maxres = (8192, 8192)),
+    hevc_10 = dict(maxres = (4096, 4096)),
+    vp8     = dict(maxres = (4096, 4096)),
+    vp9_8   = dict(maxres = (4096, 4096)),
+    vp9_10  = dict(maxres = (4096, 4096)),
+  ),
+  encode  = dict(
+    jpeg    = dict(maxres = (8192, 8192)),
+    mpeg2   = dict(maxres = (4096, 4096)),
+    avc     = dict(maxres = (4096, 4096)),
+    hevc_8  = dict(maxres = (8192, 8192)),
+    hevc_10 = dict(maxres = (4096, 4096)),
+    vp8     = dict(maxres = (4096, 4096)),
+    vp9_8   = dict(maxres = (4096, 4096)),
+    vp9_10  = dict(maxres = (4096, 4096)),
```

Comment:
vp9 encoding is not supported in normal mode on ICL

Reply:
Yes, good catch.  I think I read the media driver feature E vs. Es backwards :)... will fix.